plugins/by-name/image/default.nix

```
@@ -1,37 +1,53 @@
 {
   lib,
-  helpers,
-  config,
   pkgs,
   ...
 }:
-with lib;
 let
-  cfg = config.plugins.image;
+  inherit (lib) types;
+  inherit (lib.nixvim) defaultNullOpts;
 in
-{
-  meta.maintainers = [ maintainers.GaetanLepage ];
-
-  options.plugins.image = lib.nixvim.plugins.neovim.extraOptionsOptions // {
-    enable = mkEnableOption "image.nvim";
-
-    package = lib.mkPackageOption pkgs "image.nvim" {
-      default = [
-        "vimPlugins"
-        "image-nvim"
-      ];
-    };
-
+lib.nixvim.plugins.mkNeovimPlugin {
+  name = "image";
+  packPathName = "image.nvim";
+  package = "image-nvim";
+
+  maintainers = with lib.maintainers; [ GaetanLepage ];
```

Nit: I'd rather either avoid using `with` until we have multiple list items or formatting it such that adding additional items would minise the diff:

```suggestion
  maintainers = [ lib.maintainers.GaetanLepage ];
```

or

```suggestion
  maintainers = with lib.maintainers; [
    GaetanLepage
  ];
```